otx/algorithms/classification/tasks/nncf.py

```diff
@@ -15,7 +15,7 @@
 # and limitations under the License.
 
 from functools import partial
-from typing import Optional
+from typing import List, Optional
 
 import otx.algorithms.classification.adapters.mmcls.nncf.patches  # noqa: F401  # pylint: disable=unused-import
```

Comment:
Can't we delete unused import?

Reply:
`patches` was not intended to be used in anywhere as normal modules but to patche mm libraries to work with NNCF.
It is not used explicitly in here but certainly it is needed.

Reply:
I see, can't it be tasks.__init__.py right?

Reply:
Do you mean that can we put this import in `task/__init__.py`?

Reply:
Yes!

Reply:
We could but in that case, it is less obvious that we are patching functions for NNCF task.
In fact, I wanted `NNCFTask` not to be included in `task/__init__.py` to avoid bugs caused by `patches` for other tasks.

Reply:
Okay I understand